### ISSUE TITLE
JW Platform.com (partial) breaks video

### ISSUE THREAD

**Issue by terrorist96** [EFForg/https-everywhere#4531]
http://dailycaller.com/2016/03/30/jane-sanders-hillary-tries-to-get-votes-from-people-at-the-worst-time-in-their-life-video/

JW Platform.com (partial) breaks video

**Comment by funkydude** [comment]
Looking at this ruleset, I think it should just be deleted.

`content.` could cause issues on any number of sites since we're talking JWPlayer here.
`dashboard.` doesn't seem to be used anymore and forwards to `account.jwplayer.com` which is covered in the LongTail ruleset anyway.

@J0WI ?

**Comment by J0WI** [comment]
We can try to exclude the xml file to work around the cors bugs.
`api.jwplatform.com` is also missing, but in case we have to delete the rule we should merge them in LongTail.

**Comment by terrorist96** [comment]
Another example: http://www.wnd.com/2016/04/trump-erupts-as-cruz-sweeps-colorado-without-votes/

breaks both videos. First video is **also** broken by Google API. #4545

**Comment by numismatika** [comment]
Has anyone tried to contact JW and ask them if they can't enable cors on their side to fix it?

**Comment by J0WI** [comment]
Maybe someone official from the EFF could do that (@jsha).
Mostly the won't care about daily requests.

**Comment by jsha** [comment]
In general, when rulesets, especially partial rulesets, are broken, I think we should just disable or delete them. In this case it sounds like there's a specific video provider (jwplatform) that is causing a lot of rules to break that used to work. Is that an accurate statement? @Hainish can you take a look?

**Comment by terrorist96** [comment]
Still broken in Version: 2016.5.10

**Comment by J0WI** [comment]
I made an exclusion for the affected xml files in #4787.
I didn't remove `dashboard` because it still works and there are some deep links that doesn't redirect.

**Comment by terrorist96** [comment]
@J0WI Still broken in 2016.6.9.

**Comment by J0WI** [comment]
This has been fixed, your example depends on #4545 which indeed hasn't been fixed yet.

**Comment by terrorist96** [comment]
Nope. Disabling Google APIs and leaving JW active, the videos are still broken in both links.

**Comment by J0WI** [comment]
Works for me with on Firefox a clean profile out of the box.

**Comment by terrorist96** [comment]
And it doesn't work for me on Chrome, in a clean profile, with only HTTPS Everywhere installed.

**Comment by J0WI** [comment]
Please open a new issue for that.
